File: C Programming/problem on numbericals/problem17.c
/*
17.Write a program which accept N and print first 5 multiples of N. 
	Author : Milind Nalawade
	
	Date : 25/06/2021
	
	Language : C language
	
*/

#include<stdio.h>

//Logic

void Print(int no)
{
	int i=1;
	while(i<=no)
	{
		printf("%d ",i*5);
		i++;
	}
}

// Entry point function
int main()
{
	int i=0;
	printf("Enter your number : ");
	scanf("%d",&i);
	Print(i);
	return 0;
}

File: Python Programming/problems on numbericals/problem17.py
'''
17.Write a program which accept N and print first 5 multiples of N. 
	
	Author : Milind Nalawade
	
	Date : 25/06/2021
	
	Language : Python 3.9.0

'''
#====
#Logic
#====
def Print(no):
	i=1
	while i<=no:
		print(i*5,end=" ")
		i=i+1
		
#=============
#Entry point function
#=============
def main():
	ino=int(input("Enter your number : "))
	Print(ino)
#=======
#starter
#=======
if __name__=="__main__":
	main()